src/app/sitemap.ts

```diff
@@ -0,0 +1,23 @@
+import { CATEGORIES, SITE_URL } from '@/constants';
+import { MetadataRoute } from 'next';
+
+export default function sitemap(): MetadataRoute.Sitemap {
+    const staticPaths = ['', 'credits', 'jobs'];
+
+    const dynamicPaths = [
+        ...CATEGORIES.map((category) => {
+            return category.url;
```

Comment:
Change this to `category.path` now, to remove the double `/`

Reply:
done